forms/preside-objects/email_content_library_content.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<form i18nBaseUri="preside-objects.content_library_content:">
```

Comment:
should use the feature flag on the form definition

Reply:
feature flag added